libraries/rush-lib/src/cli/RushXCommandLine.ts

```diff
@@ -223,7 +223,30 @@ export class RushXCommandLine {
             pnpmSyncJsonPath,
             ensureFolder: FileSystem.ensureFolderAsync,
             forEachAsyncWithConcurrency: Async.forEachAsync,
-            getPackageIncludedFiles: PackageExtractor.getPackageIncludedFilesAsync
+            getPackageIncludedFiles: PackageExtractor.getPackageIncludedFilesAsync,
+            logMessageCallback: (logMessageOptions: ILogMessageCallbackOptions) => {
+              const { message, messageKind } = logMessageOptions;
+              switch (messageKind) {
+                case 'error':
+                  // eslint-disable-next-line no-console
+                  console.error(Colorize.red(message));
+                  break;
+                case 'warning':
+                  // eslint-disable-next-line no-console
+                  console.error(Colorize.yellow(message));
+                  break;
+                case 'verbose':
+                  if (rushxArguments.isDebug) {
+                    // eslint-disable-next-line no-console
+                    console.error(message);
+                  }
+                  break;
+                default:
+                  // eslint-disable-next-line no-console
+                  console.log(message);
+                  break;
```

Comment:
Can you use the `Terminal` API in this file? You don't have to convert everything, but it'd be good to get that started.